src/maxdiffusion/configs/base_2_base.yml

```diff
@@ -19,6 +19,7 @@ metrics_file: "" # for testing, local file that stores scalar metrics. If empty,
 write_metrics: True
 gcs_metrics: False
 metrics_period: 100  # calculate step time and loss
+metrics_buffer: false
```

Comment:
@aireenmei can we resue write_metrics flag? if write_metrics set, we will do ping-pong metrics out. if not, no metrics file will be generated.

Reply:
I currently use write_metrics=false to turn off any step time logging (including mllog) to get a clean baseline, that help access any logging overhead. See my comments https://buganizer.corp.google.com/issues/343225248#comment6

Reply:
we already got two flags for metrics write out, `write_metrics`, `gcs_metrics`, we can apply buffer metrics as default for metrics write out. might not need to introduce new flag

Reply:
done